clients/extension/scripts/popup.js

```diff
@@ -13,14 +13,7 @@
  * Lighthouse itself, mapped to more useful strings to report to the user.
  */
 const NON_BUG_ERROR_MESSAGES = {
-  // The user tries to review an error page or has network issues
-  'ERRORED_DOCUMENT_REQUEST': 'Unable to load the page. Please verify the url you ' +
-      'are trying to review.',
-  'FAILED_DOCUMENT_REQUEST': 'Unable to load the page. Please verify the url you ' +
```

Comment:
this should still be possible, shouldn't it? e.g. for https://expired.badssl.com/

Reply:
Not as a fatal error. The popup only catches fatal errors and replaces the bug prompt with these messages. `FAILED_DOCUMENT_REQUEST` hasn't been fatal for quite some time.